src/NodaTime.Demo/IntervalDemo.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright 2019 The Noda Time Authors. All rights reserved.
+// Use of this source code is governed by the Apache License 2.0,
+// as found in the LICENSE.txt file.
+
+using NUnit.Framework;
+
+namespace NodaTime.Demo
+{
+    public class IntervalDemo
+    {
+        [Test]
+        public void Construction()
+        {
+            Instant start = Instant.FromUtc(2019, 1, 1, 15, 25, 48);
+            Instant end = Instant.FromUtc(2019, 1, 1, 16, 25, 48);
+            Interval interval = Snippet.For(new Interval(start, end));
+            Assert.AreEqual(start, interval.Start);
+            Assert.AreEqual(end, interval.End);
+        }
+
+        [Test]
+        public void Construction_WithNoBeginning()
+        {
+            Instant end = Instant.FromUtc(2019, 1, 1, 16, 25, 48);
+            Interval interval = Snippet.For(new Interval(null, end));
+            Assert.False(interval.HasStart);
```

Comment:
I don't currently support Assert.False. I can add that though.

Reply:
Well, I guess I could just use `Assert.True` and negate the expression. Would that work?

Reply:
No need - I've added Assert.False support. That'll make it simpler - otherwise it would end up with `Console.WriteLine(!interval.HasStart)` which would be weird to read.

It might be worth you getting set up to generate the snippets locally and look at them before submitting a PR. That involves cloning the nodatime.org repo as a peer of the nodatime repo, and then running the snippet generator - I'm not sure whether I've got a script to just generate the snippets at the moment, but I could write one.